src/cogs/esports/events/tags.py

```diff
@@ -111,3 +111,18 @@ async def on_eztag_msg(self, message: discord.Message):
             if eztag.delete_after:
                 self.bot.loop.create_task(delete_denied_message(message, 60))
                 self.bot.loop.create_task(delete_denied_message(msg, 60))
+
+    @Cog.listener(name="on_guild_channel_delete")
+    async def on_channel_delete(self, channel: discord.TextChannel):
```

Comment:
I don't think, dispatch works in that way, instead why not checking channel type inside the function
```py
async def on_channel_delete(self, channel: discord.abc.GuildChannel) -> None:
    if not isinstance(channel, discord.TextChannel):
        return
    ...
```

Reply:
Updated the PR